src/state/userInput/__tests__/userInputReducer.test.ts

```diff
@@ -0,0 +1,27 @@
+import userInputSlice, {
+  userInputInitialState,
+  userInputActions,
+} from '../userInputReducer'
+
+describe('userInputSlice', () => {
+  it('initialize slice with initialValue', () => {
```

Comment:
Er dette en nødvendig test? Det virker som at vi tester at `createSlice` greier å sette `initialState`. Er det meningen at dette er en slags regresjonstest for å passe på at ingen utviklere ved et uhell setter `initialState` til noe annet?

Reply:
Ok enig 👍  Liker godt at testen er liten og ikke trenger å mocke masse greier